e2e/testmain_test.go

```diff
@@ -74,10 +80,25 @@ func TestMain(m *testing.M) {
 	parseFlagsOrDie()
 
 	ctx := context.Background()
-	client, err := google.DefaultClient(ctx, compute.ComputeScope)
+
+	credentials, err := google.FindDefaultCredentials(ctx, compute.ComputeScope)
 	if err != nil {
 		log.Fatal(err)
 	}
+	ts := credentials.TokenSource
+
+	if testFlags.serviceAccountName != "" {
```

Comment:
Can you add comment what credentials will be taken if this field is empty

Reply:
the behavior stays the same as before